.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     executor:
       name: node/default
       # The Node version here must be kept in sync with that in `package.json`.
-      tag: '12.14.1'
+      tag: '18.18.0'
```

Comment:
you can move to v22 later too
(in follow-ups)

Reply:
Yes, in follow-ups